lib/fluent/plugin/in_syslog.rb

```diff
@@ -107,6 +107,11 @@ class SyslogInput < Input
       config_param :with_priority, :bool, default: true
     end
 
+    # overwrite server plugin to change default to :udp
+    config_section :transport, required: false, multi: false, init: true, param_name: :transport_config do
+      config_argument :protocol, :enum, list: [:tcp, :tls], default: :udp
```

Comment:
Should we use the following line?

```ruby
config_argument :protocol, :enum, list: [:tcp, :udp, :tls], default: :udp
```

Reply:
Right. Thank you. https://github.com/fluent/fluentd/pull/2399/commits/bf319e95b3f101843c7cfd143c80b7ce2d8df38b